config/webpack/webpack.config.base.js

```diff
@@ -41,28 +52,20 @@ module.exports = (env) => {
     },
   };
 
-  const FILE_LOADER_ASSETS_IMAGES = {
-    test: /\.(gif|ico|jpg|jpeg|png|svg|webp)(\?.*)?$/,
-    exclude: /node_modules/,
-    use: [{
-      loader: 'file-loader',
-      options: {
-        name: '[name].[hash:8].[ext]',
-        outputPath: `${APP_PATHS.REL.STATIC_ASSETS_IMAGES}/`,
-      }
-    }]
-  };
-
-  /*
-   * plugins
-   */
+  //
+  // plugins
+  //
 
-  const BANNER_PLUGIN = new Webpack.BannerPlugin({
-    banner: APP_CONFIG.BANNER,
+  const BANNER_PLUGIN = new webpack.BannerPlugin({
+    banner: BANNER,
     entryOnly: true,
   });
 
-  const DEFINE_PLUGIN = new Webpack.DefinePlugin({
+  const PROVIDE_PLUGIN = new webpack.ProvidePlugin({
+    process: 'process/browser'
+  });
```

Comment:
In theory, this won't be necessary once this is released:

https://github.com/openlattice/lattice-auth/pull/861